test/Fencer/Server/Test.hs

```diff
@@ -87,32 +87,53 @@ test_serverResponseNoRules =
 -- gRPC server
 ----------------------------------------------------------------------------
 
+-- | A type combining a logger, a handle for the logging file, a
+-- thread id and an application state.
+type LogIdSt = (Logger.Logger, Handle, ThreadId, AppState)
```

Comment:
I'd make it a `data` with named fields, but no big deal.

Reply:
Makes sense. This crossed my mind. It's true it got big enough to deserve its own type.